crds/main.go

```diff
@@ -0,0 +1,374 @@
+// Copyright 2016-2020, Pulumi Corporation.
```

Comment:
Let's move these files to the `provider/cmd` directory. https://github.com/pulumi/pulumi-kubernetes/tree/master/provider/cmd